Helper/PaymentMethods.php

```diff
@@ -474,6 +479,18 @@ protected function getPaymentMethodsRequest(
                 $this->adyenHelper->padShopperReference($this->getCurrentShopperReference());
         }
 
+        $quote = $this->checkoutSession->getQuote();
```

Comment:
There is already a `$quote` argument in the method signature. Can we leverage that instead of injecting the `CheckoutSession`?